drawee/src/main/java/com/facebook/drawee/drawable/ScalingUtils.java

```diff
@@ -415,10 +427,16 @@ public Matrix getTransform(
         int childHeight,
         float focusX,
         float focusY) {
-      mScaleTypeFrom.getTransform(transform, parentBounds, childWidth, childHeight, focusX, focusY);
-      transform.getValues(mMatrixValuesFrom);
-      mScaleTypeTo.getTransform(transform, parentBounds, childWidth, childHeight, focusX, focusY);
-      transform.getValues(mMatrixValuesTo);
+      if (mMatrixValuesFrom == null) {
```

Comment:
I am a bit concerned about this as there may be some correctness issues. For example, if the child dimensions change in the middle of the animation this would cause a display issue. Let's just leave this computation unconditionally as it was before.

Context:
Child bounds indeed may change in the middle of the animation. Fresco allows specifying low-res and high-res image when building a Drawee controller, and once the high-res image arrives it will automatically replace the previously displayed low-res image. Low-res and high-res images are exactly the same except that the low-res image is a downscaled version of the high-res one. The content and the aspect ratio are exactly the same, but width and height are not. With a proper scale type (e.g. center_crop or fit_center) this swap will go completely smooth, but for that to happen scale type always needs to recompute the transform.